components/usage-api/usage/v1/billing.proto

```diff
@@ -70,3 +77,24 @@ message CreateStripeCustomerRequest {
 message CreateStripeCustomerResponse {
   StripeCustomer customer = 1;
 }
+
+message CreateStripeSubscriptionRequest {
+  string attribution_id = 1;
+  string stripe_intent_id = 2;
+
+  int32 usage_limit = 3;
+}
+
+message CreateStripeSubscriptionResponse {
+  StripeCustomer customer = 1;
+  v1.CostCenter cost_center = 2;
+}
```

Comment:
I'd have expected to get a `StripeSubscription` in the response. What's the reasoning we're not returning one? Do we not need it?

We can add it later as well.